lib/as_action_ocaml.ml

```diff
@@ -67,7 +67,7 @@ let compute_deps_ml = compute_deps `Ml
 
 let compile_mli
     ?(needs = []) ?(pkgs = nop) ?(args = []) ~ocamlc ~annot ~incs ~src () =
-  let ctx = As_ctx.v [`OCaml; `Compile; `Src `Mli] in
+  let ctx = As_ctx.v [`OCaml; `Compile; `Src `Mli; `Target `Byte] in
```

Comment:
Maybe the most future proof of solving that problem is to add a `target` argument to this function with either `` `Target `Byte`` or `` `Target `Native` that you add to the context according to which `ocamlc` is passed as an argument (i.e. define that [here](https://github.com/samoht/assemblage/blob/master/lib/as_part_unit.ml#L128-L129)).